### PR TITLE
Chore model extraction

### PR DESCRIPTION
**Pull request by flbulgarelli**
~:warning: Work In progress~ Done & ready to merge

Related to https://github.com/mumuki/mumukit-platform/pull/24
Related to https://github.com/mumuki/mumukit-core/pull/8

Pending: 

- [x] Tasks Cleanup
- [x] Licenses cleanup
- ~[ ] Robots clenaup~ won't fix in this PR

### CODE REVIEW

File: lib/mumuki/laboratory.rb [hunk 1]
```diff
@@ -9,23 +9,41 @@ module Laboratory
...
+Mumukit::Platform.configure do |config|
+  config.application = Mumukit::Platform.laboratory
+  config.web_framework = Mumukit::Platform::WebFramework::Rails
+end
+
+class Mumuki::Laboratory::Engine < ::Rails::Engine
+  config.i18n.available_locales = Mumukit::Platform::Locale.supported
+end
+
+module Mumukit::Platform::OrganizationMapping::Path
```
Comment (julian-berbel) [inline lib/mumuki/laboratory.rb]: I think this block is no longer necessary :thinking:

Comment (julian-berbel) [reply lib/mumuki/laboratory.rb]: uhhhhh seems like it was necessary after all! :sweat_smile: removing it broke login in this branch, so we should restore it before merging :warning:

Comment (luchotc) [reply lib/mumuki/laboratory.rb]: Yeah, but i'd like to see hyper being called here :tada:

File: domain/db/migrate/20181105152420_remove_contact_email_from_book.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RemoveContactEmailFromBook < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :books, :contact_email
```
Comment (luchotc) [inline domain/db/migrate/20181105152420_remove_contact_email_from_book.rb]: We should add the type to make it reversible

File: lib/events.rb [hunk 3]
```diff
@@ -2,33 +2,26 @@
...
-    Exam.import_from_json! body
+    Exam.import_from_resource_h! body
   end
 
   # Emitted by assigment manual evaluation in classroom
   event 'AssignmentManuallyEvaluated' do |payload|
     Assignment.evaluate_manually! payload.deep_symbolize_keys[:assignment]
   end
-
-  # Emitted by content creation and modification in bibliotheca
```
Comment (flbulgarelli) [inline lib/events.rb]: We should recover this block

**Comment by luchotc** [comment]
:100: :+1:

**Pull request merged**